youthful-piety/modernized/01-preface.adoc

```diff
@@ -2,7 +2,7 @@
 ____
 Honourable age is not that which stands in length of time,
 nor that is measured by number of years, but wisdom is the gray hair unto men,
-and an unspotted life is old age.
+and an unspotted life is old age. -Wisdom
```

Comment:
this quote (which I think is taken from the Wisdom of Solomon) didn't need you to put "--Wisdom" after it, becase the reference to Wisdom is already above in the epigraph tag: [quote.epigraph, , Wisdom].   So on the PDF it said Wisdom twice :) see pic.  
![image](https://user-images.githubusercontent.com/26820297/81701058-7d07d980-9437-11ea-98db-95d07d1e8c8b.png)

Reply:
ahhh!!! I didn't even notice that!! Ugh.